docs/pages/setup/reference/metrics.mdx

```diff
@@ -109,6 +109,9 @@ Now you can see the monitoring information by visiting several endpoints:
 | `rx` | counter | Teleport | Number of bytes received. |
 | `server_interactive_sessions_total` | gauge | Teleport | Number of active sessions. |
 | `teleport_build_info` | gauge | Teleport | Provides build information of Teleport including gitref (git describe --long --tags), Go version, and Teleport version. The value of this gauge will always be 1. |
+| `teleport_registered_servers` | gauge | Teleport Auth | The number of Teleport servers (a server consists of one or more Teleport services) that have connected to the Teleport cluster, including the Teleport version. 
+
+After disconnecting, a Teleport server has a TTL of 10 minutes, so this value will include servers that have recently disconnected but have not reached their TTL. |
```

Comment:
As it turns out, adding a new paragraph to a cell in a Markdown table breaks the rendering of the rest of the table. I'll create an issue for this, and in the meantime we should remove the paragraph break.